noir/noir-repo/noir_stdlib/src/embedded_curve_ops.nr

```diff
@@ -42,22 +48,22 @@ impl Neg for EmbeddedCurvePoint {
 #[foreign(multi_scalar_mul)]
 // docs:start:multi_scalar_mul
 pub fn multi_scalar_mul<N>(
-    points: [Field; N], // points represented as x and y coordinates [x1, y1, x2, y2, ...]
-    scalars: [Field; N] // scalars represented as low and high limbs [low1, high1, low2, high2, ...]
-) -> [Field; 2]
+    points: [EmbeddedCurvePoint; N],
+    scalars_lo: [Field; N], // scalars represented as low and high limbs [low1, low2, low3, ...]
+    scalars_hi: [Field; N] // [high1, high2, high3,...]
```

Comment:
I think it would be nicer to wrap these up into an `EmbeddedCurveScalar`, emphatically not a blocker for this PR however as I don't want to delay this.

Can you open an issue to add this struct to the stdlib at some point?

Reply:
Considering how annoying it is to get PRs changing ACIR format merged, I am tempted to add this in this PR

Reply:
Go for it, I was wanting to avoid forcing making this change as it's relatively late in the review cycle for this PR but happy to roll it in.

Reply:
Done